### DISCUSSION TITLE
How do I set Audiences to target on Navigation Link

### DISCUSSION THREAD

**Discussion by khinklenj** [pnp/pnpcore#1064] [Q&A] [answered]
How do I set Audiences to target on Navigation Link in code. 

![image](https://user-images.githubusercontent.com/6447184/208818717-9236b332-9ffd-4966-a095-4b27b11e442c.png)

![image](https://user-images.githubusercontent.com/6447184/208818605-8f8c1ba7-f599-408c-8d1f-3faa665deace.png)

**Comment by jansenbe** [comment, marked as answer]
@khinklenj : I've created issue #1065 to track this, currently we do not have support for doing this.

**Reply by jansenbe** [reply]
Feature has been implemented.
